eclipse/plugin/src/nl/tudelft/watchdog/eclipse/logic/ui/WatchDogEventManager.java

```diff
@@ -1,56 +0,0 @@
-package nl.tudelft.watchdog.eclipse.logic.ui;
-
-import org.eclipse.ui.texteditor.ITextEditor;
-
-import nl.tudelft.watchdog.core.logic.document.Document;
-import nl.tudelft.watchdog.core.logic.document.EditorWrapperBase;
-import nl.tudelft.watchdog.core.logic.interval.IDEIntervalManagerBase;
-import nl.tudelft.watchdog.core.logic.interval.intervaltypes.PerspectiveInterval;
-import nl.tudelft.watchdog.core.logic.ui.WatchDogEventManagerBase;
-import nl.tudelft.watchdog.core.logic.ui.events.WatchDogEvent;
-import nl.tudelft.watchdog.core.logic.ui.events.WatchDogEvent.EventType;
-import nl.tudelft.watchdog.eclipse.logic.document.DocumentCreator;
-import nl.tudelft.watchdog.eclipse.logic.document.EditorWrapper;
-import nl.tudelft.watchdog.eclipse.logic.interval.intervaltypes.JUnitInterval;
-
-/**
```

Comment:
Can we preserve this comment somehow (I think it best fits in the process method)?

Reply:
Done :+1: